javascript-view-swapping/main.js

```diff
@@ -0,0 +1,32 @@
+const $tabContainer = document.querySelector('.tab-container');
+const $tabElements = document.querySelectorAll('.tab');
+const $viewElements = document.querySelectorAll('.view');
+
+function tabContainerHandler(event) {
+  for (let i = 0; i < $tabElements.length; i++) {
+    if (event.target.matches('.tab') && event.target === $tabElements[i]) {
+      event.target.className = 'tab active';
+    } else if (
+      event.target.matches('.tab') &&
+      event.target !== $tabElements[i]
+    ) {
+      $tabElements[i].className = 'tab';
+    }
```

Comment:
We've already checked for both of these conditionals at this point, you can just do `else`